R/logging.R

```diff
@@ -120,7 +120,7 @@ log_problems <- function(notes, control, split, loc, res, bad_only = FALSE) {
 
   wrn <- res$signals
   if (length(wrn) > 0) {
-    wrn_msg <- purrr::map_chr(wrn, ~ .x$message)
+    wrn_msg <- purrr::map_chr(wrn, ~conditionMessage(.x))
```

Comment:
This was the underlying issue with https://github.com/tidymodels/finetune/pull/50. This function is called in the code path for handling logging cases with warnings but no errors. The condition message, in the case of the lifecycle message raised there (which is why this only occurred in `test()`—lifecycle messages aren't raised interactively when not called by the user), had length greater than 2. `map_chr()` thus errored out, which caused `log_problems()` to error out, which is not inside of the `.catch_and_log` code path that effectively handles user errors.